pkg/apis/servicecatalog/types.go

```diff
@@ -309,4 +309,15 @@ const (
 	BindingConditionReady BindingConditionType = "Ready"
```

Comment:
What is the state when the binding has been created but its not been populated by the SB yet?